test/components/publication_header_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'component_test_helper'
+
+class PublicationHeaderTest < ComponentTestCase
+  def component_name
+    "publication-header"
+  end
+
+  test "fails to render a publication-header when no text is given" do
```

Comment:
> when no text is given
Should this be title? title and context?
It looks like all properties are required?